spec/index.bs

```diff
@@ -6763,15 +6762,17 @@ following members:
                                                         must be 1.
                                             </dl>
                                 </dl>
+
```

Comment:
I'm a little confused: this change puts the "if" outside the compatmode div, while some changes below put it outside. Should/can we be consistent? (I do recall I had to do this in some cases to appease Bikeshed, so if that's the case feel free to ignore.)